src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/UpdateSubmission.java

```diff
@@ -92,6 +96,14 @@ public void setEntityNumber(String entityNumber) {
         this.entityNumber = entityNumber;
     }
 
+    public String getExistingEntityName() {
```

Comment:
This needs a @JsonProperty(ENTITY_NAME)

Can we name this field just entityName (drop existing). 
So public static final String ENTITY_NAME = "entityName";

Reply:
By "needs" - to be consistent with other decorated fields.

Reply:
ah okay, did not want to get it mixed up with the other entity_name field in userSubmission but if that's preferred i'll make the change

Reply:
will make the change on xml side too